onnxruntime/core/providers/cpu/ml/treeregressor.h

```diff
@@ -8,12 +8,16 @@ namespace onnxruntime {
 namespace ml {
 template <typename T>
 class TreeEnsembleRegressor final : public OpKernel {
+  typedef T TI;  // input type
```

Comment:
nit: `using TI = T` (see [https://google.github.io/styleguide/cppguide.html#Aliases](https://google.github.io/styleguide/cppguide.html#Aliases))
also if it was named `InputType` we wouldn't need a comment :)

Reply:
I renamed everything in the impacted files: TI -> InpuyType, TH -> ThresholdType, TO -> OutputType

Reply:
FYI, noticed there are some TI/TH/TO references left in the comments